### DISCUSSION TITLE
Send weekly reports - criteria to run

### DISCUSSION THREAD

**Discussion by QtaAbc-adocao** [sheltermanager/asm3#1506] [Q&A / Help] [answered]
Hello, 

I am trying to set a weekly report to an email address and it keeps appearing the same error: "This report cannot be sent by email as it requires criteria to run.". Can you help me please?

Kind regards

**Comment by Tetzor** [comment, marked as answer]
Hi,

I'm afraid it isn't possible to run a report on a schedule if that report requires criteria such as dates to be set. Reports with criteria can only be run manually after selecting the criteria. 

Some of the reports have alternate versions available which don't require criteria and use set dates such as "In/Out Today". If the report you are trying to email doesn't have an alternative available, you could edit the criteria in Settings->Reports. 

Jon

**Reply by QtaAbc-adocao** [reply]
Hi,

Thank you for the reply.
Deleting the criteria from the report corrects the error but when I add the "Email To", "When", and "at", nothing happens. No e-mail is sent despite having data.

Can you help me please?

Kind regards

**Comment by Tetzor** [comment]
Hi,

Please could you contact us via our support form with your account details and the name of the report so we can take a look for you?.

https://sheltermanager.com/site/en_support.html

Jon
